products-1.html

```diff
@@ -33,93 +33,67 @@
-                            <p>Price</p>
+                            <div class="image"><a href="https://www.farmers.co.nz/the-ordinary-niacinamide-10-zinc-1-30ml-6394630"><img src="images/oily/to_niacinamide_large.jpg"></a></div>
+                            <a class="h2" href="https://www.farmers.co.nz/the-ordinary-niacinamide-10-zinc-1-30ml-6394630">The Ordinary 10% Niacinamide Serum + Zinc</a>
+                            <p>$12.80 NZD</p>
                         </div>
 
                         <div class="product-box">
-                            <div class="image"><a href="#"><img src="placeholder.png"></a></div>
-                            <a class="h2" href="#">Product name</a>
-                            <p>Price</p>
+                            <div class="image"><a href="https://www.sephora.nz/products/first-aid-beauty-coconut-water-cream/v/default"><img src="images/oily/fa_coconutcream.jpeg"></a></div>
+                            <a class="h2" href="https://www.sephora.nz/products/first-aid-beauty-coconut-water-cream/v/default">First Aid Beauty Coconut Water Cream</a>
+                            <p>$63 NZD</p>
 
                         </div>
 
                         <div class="product-box">
-                            <div class="image"><a href="#"><img src="placeholder.png"></a></div>
-                            <a class="h2" href="#">Product name</a>
-                            <p>Price</p>
+                            <div class="image"><a href="https://nz.iherb.com/pr/some-by-mi-aha-bha-pha-30-days-miracle-cream-60-g/95241"><img src="images/oily/sbm_miraclecream_large.jpg"></a></div>
+                            <a class="h2" href="https://nz.iherb.com/pr/some-by-mi-aha-bha-pha-30-days-miracle-cream-60-g/95241">Some By Mi AHA/BHA/PHA Miracle Cream</a>
+                            <p>$28.60 NZD</p>
 
                         </div>
 
                         <div class="product-box">
-                            <div class="image"><a href="#"><img src="placeholder.png"></a></div>
-                            <a class="h2" href="#">Product name</a>
-                            <p>Price</p>
+                            <div class="image"><a href="https://amp.ladybonita.co.nz/products/purito-comfy-water-sun-block-spf50-pa"><img src="images/oily/purito_sunblock_large.jpg"></a></div>
+                            <a class="h2" href="https://amp.ladybonita.co.nz/products/purito-comfy-water-sun-block-spf50-pa">Purito Comfy Water Sun Block</a>
+                            <p>$29.99 NZD</p>
 
                         </div>
                     </div>
 
                     <div class="product-box" id="first">
-                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
-                            <a class="h2" href="#">Product name</a>
-                            <p>Price</p>
-
-                    </div>
-
-                    <div class="product-box">

```

Comment:
In version 7, I removed the last row of boxes as I thought finding 12 products would take too long, so I cut it down to 8 products instead.